<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>键盘操作（单击输入框然后按下对应键）</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>上</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="upKey"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>下</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="downKey"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>左</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leftKey"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>右</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rightKey"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>动作</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="actionKey"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cancelKey"/>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>菜单/战斗</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="menuKey"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setDefaultButton">
      <property name="text">
       <string>全部设成默认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>动作和取消键相当于模拟器上的A键和B键，菜单/战斗键相当于模拟器上的start键，用于呼出菜单或者开始一场战斗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
